demo/src/app/components/+accordion/demos/dynamic-body/dynamic-body.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-accordion-dynamic-body',
+  templateUrl: './dynamic-body.html'
+})
+export class DemoAccordionDynamicBodyComponent {
+  items: string[] = ['Item 1', 'Item 2', 'Item 3'];
```

Comment:
same
if type annotation isn't provided, typescript will use the type of the first assigned value